src/PrestaShopBundle/Form/Exception/DataProviderException.php

```diff
@@ -29,26 +29,27 @@
 namespace PrestaShopBundle\Form\Exception;
 
 use PrestaShop\PrestaShop\Core\Domain\Exception\DomainException;
+use PrestaShop\PrestaShop\Core\Form\ErrorMessage\ConfigurationErrorCollection;
 use Throwable;
 
 class DataProviderException extends DomainException
 {
     /**
-     * @var InvalidConfigurationDataErrorCollection
+     * @var ConfigurationErrorCollection
      */
-    private $InvalidConfigurationDataErrors;
+    private $configurationErrors;
 
-    public function __construct($message = '', $code = 0, Throwable $previous = null, ?InvalidConfigurationDataErrorCollection $InvalidConfigurationDataErrors = null)
+    public function __construct($message = '', $code = 0, Throwable $previous = null, ?ConfigurationErrorCollection $configurationErrors = null)
     {
         parent::__construct($message, $code, $previous);
-        $this->InvalidConfigurationDataErrors = $InvalidConfigurationDataErrors ?: new InvalidConfigurationDataErrorCollection();
+        $this->configurationErrors = $configurationErrors ?: new ConfigurationErrorCollection();
     }
 
     /**
-     * @return InvalidConfigurationDataErrorCollection
+     * @return ConfigurationErrorCollection
      */
-    public function getInvalidConfigurationDataErrors(): InvalidConfigurationDataErrorCollection
+    public function getConfigurationErrors(): ConfigurationErrorCollection
     {
```

Comment:
I guess this also considered BC break, should I change it back for the sake of simplicity?